File: atcoder/atcoder_problems/abc/abc101~200/abc101~150/abc136/abc136_b.py
N = int(input())

ans = 0
for i in range(1, N+1):
    if 1 <= i <= 9:
        ans += 1
    if 100 <= i <= 999:
        ans += 1
    if 10000 <= i <= 99999:
        ans += 1

print(ans)

# N = int(input())

# ans = 0
# for i in range(1,N+1):
#     s = str(i)
#     if len(s) % 2 == 1:
#         ans += 1

# print(ans)

File: atcoder/atcoder_problems/abc/abc101~200/abc101~150/abc136/abc136_b.cpp
#include <bits/stdc++.h>
using namespace std;
#include <atcoder/all>
using namespace atcoder;
#define rep(i,n) for (int i = 0; i < (n); ++i)

bool f(int x) {
    int kata = 0;
    while (x) {
        kata++;
        x /= 10;
    }
    return kata % 2 == 1;
}

int main() {
    int N;
    cin >> N;
    int ans = 0;
    for (int i = 1; i <= N; i++) {
        if (f(i)) ans++;
    }
    cout << ans << endl;
    return 0;
}



// #include <bits/stdc++.h>
// using namespace std;
// #include <atcoder/all>
// using namespace atcoder;
// #define rep(i,n) for (int i = 0; i < (n); ++i)

// int count(int n) {
//     int cnt = 0;
//     while (n) {
//         cnt += 1;
//         n /= 10;
//     }
//     return cnt;
// }

// int main() {
//     int N;
//     cin >> N;
//     int ans = 0;
//     for (int i = 1; i <= N; i++) {
//         if (count(i) % 2 == 1) ans++;
//     }
//     cout << ans << endl;
//     return 0;
// }